File: untold_comparison_programs/02_variables/go/main.go
package main

import "fmt"

func main() {
    name := "Untold"
    fmt.Printf("Name: %s\n", name)

    version := 2.1
    fmt.Printf("Version: %v\n", version)

    isAwesome := true
    if isAwesome {
        fmt.Println("Untold is awesome!")
    }

    const PI = 3.14159
    fmt.Printf("PI: %v\n", PI)

    numbers := []int{1, 2, 3, 4, 5}
    fmt.Printf("Numbers: %v\n", numbers)
    fmt.Printf("Count: %d\n", len(numbers))

    data := map[string]string{"name": "Untold", "version": "2.1"}
    fmt.Printf("Data: %v\n", data)
    fmt.Printf("Get name: %s\n", data["name"])
}

File: untold_comparison_programs/02_variables/javascript/main.js
// Variables & Data Types
let name = "Untold";
console.log("Name: " + name);

let version: number = 2.1;
console.log("Version: " + version);

let isAwesome: boolean = true;
if (isAwesome) {
    console.log("Untold is awesome!");
}

const PI = 3.14159;
console.log("PI: " + PI);

let numbers = [1, 2, 3, 4, 5];
console.log("Numbers: " + numbers);
console.log("Count: " + numbers.length);

let data = {name: "Untold", version: "2.1"};
console.log("Data: " + JSON.stringify(data));
console.log("Get name: " + data.name);